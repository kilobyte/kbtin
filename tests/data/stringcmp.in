#ali m {#if {$0} {#show {✓ $0}} #else {#show ✗ $0}}
m [ ]
m [ 1 ]
m [ a b = a b ]
m [ a b = a c ]
m [ x != x ]
m [ x == x ]
m [ x =~ x ]
m [ x =~ * ]
m [ x = * ]
m [ x !~ * ]
m [ x !~ y ]
