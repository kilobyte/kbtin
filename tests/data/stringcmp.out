#Compare   to what ? (only one var between [ ])
✗ [ ]
#Compare  1  to what ? (only one var between [ ])
✗ [ 1 ]
✓ [ a b = a b ]
✗ [ a b = a c ]
✗ [ x != x ]
✓ [ x == x ]
✓ [ x =~ x ]
✓ [ x =~ * ]
✗ [ x = * ]
✗ [ x !~ * ]
✓ [ x !~ y ]
